winter/path_parameters_argument_resolver.py

```diff
@@ -13,8 +14,13 @@ def __init__(self):
         self._url_resolver = get_resolver()
 
     def is_supported(self, argument: ComponentMethodArgument) -> bool:
-        route = get_route(argument.method)
```

Comment:
You said earlier that it's better to pass Route to this method to avoid building route. Why did you choose that way?

Reply:
I think now we can build url without getting route
if we need more than now, we will change it